packages/ts/generator-typescript-plugin-model/src/ModelSchemaProcessor.ts

```diff
@@ -272,7 +282,19 @@ export class ModelSchemaExpressionProcessor extends ModelSchemaPartProcessor<rea
     return [];
   }
 
-  #getValidatorsFromValidationConstraints(schema: ValidationConstrainedSchema): readonly Expression[] {
-    return schema['x-validation-constraints'].map(this.#parse);
+  #createValidatorsProperty(schema: Schema): PropertyAssignment | null {
+    if (!hasValidationConstraints(schema)) {
+      return null;
+    }
+
+    const constraints = schema['x-validation-constraints'].map((constraint) =>
+      this.#validationConstraintParser.parse(constraint),
+    );
+    return ts.factory.createPropertyAssignment('validators', ts.factory.createArrayLiteralExpression(constraints));
+  }
+
+  #createMetadataProperty(schema: Schema): PropertyAssignment | null {
+    const metadata = this.#metadataParser.parse(schema);
+    return metadata ? ts.factory.createPropertyAssignment('metadata', metadata) : null;
```

Comment:
Let's shrink the name a bit to have less code in the output.
```suggestion
    return metadata ? ts.factory.createPropertyAssignment('meta', metadata) : null;
```

Reply:
Done